### PR TITLE
operator-sdk: build all binaries

### PR DESCRIPTION
**Pull request by yajo**
###### Description of changes

Currently the repo provides 3 binaries. The 2 that were missing are used in upstream docker images. But to use dockerTools instead, these need to be packaged with nix!

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

@arnarg @moduon MT-1075

### CODE REVIEW

**Review by arnarg — APPROVED** [review]
Good catch! LGTM

**Review by Frostman — APPROVED** [review]

**Comment by NickCao** [comment]
Would you like to resolve the conflicts?

**Comment by yajo** [comment]
Thanks, rebased.

File: pkgs/development/tools/operator-sdk/default.nix [hunk 1]
```diff
@@ -29,9 +29,14 @@ buildGoModule rec {
...
 
   subPackages = [
+    "cmd/ansible-operator"
+    "cmd/helm-operator"
     "cmd/operator-sdk"
   ];
 
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ go ];
+
```
Comment (NickCao) [inline pkgs/development/tools/operator-sdk/default.nix:39]: Looks like these lines are redundant? (And causing eval failures)

**Comment by yajo** [comment]
El dom, 22 de ene de 2023 a las 23:26:15 PM, Nick Cao 
***@***.***> escribió:
> Looks like these lines are redundant? (And causing eval failures)
> 

Oops sorry, I didn't pay enough attention. Fixed now.

**Comment by NickCao** [comment]
@ofborg build operator-sdk

**Pull request merged**